dts/riscv/rapidsilicon/rapidsi_virgo.dtsi

```diff
@@ -0,0 +1,233 @@
+					};
+				};
+			};
+		};
+
+		flash: flash@0xB0000000 {
+			compatible = "micron,m25p32";
+			reg = <0xB0000000 DT_SIZE_M(16)>;
+			status = "disabled";
+		};
+
+		uart0: serial@0xA0420000 {
+			compatible = "ns16550";
+			reg = <0xA0420000 0xFFFF>;
+			reg-shift = <2>;
+			interrupts = <22 8>;
+			interrupt-parent = <&clic0>;
+			clock-frequency = <266666667>;
+			current-speed = <115200>;
+			status = "disabled";
+		};
+
+		pit0: pit@0xA0520000 {
+			compatible = "andestech,atcpit100";
+			reg = <0xA0520000 0xFFFF>;
+			interrupts = <19 4>;
+			interrupt-parent = <&clic0>;
+			clock-frequency = <266666667>;
+			prescaler = <600>;
+			status = "disabled";
+		};
+
+		gpio0: gpio@0xA0510000 {
+			compatible = "andestech,atcgpio100";
+			reg = <0xA0510000 0xFFFF>;
+			interrupts = <26 2>;
+			interrupt-parent = <&clic0>;
+			gpio-controller;
+			ngpios = <8>;
+			#gpio-cells = <3>;
+			status = "disabled";
+		};
+
+		i2c0: i2c@0xA0410000 {
+			compatible = "andestech,atciic100";
+			reg = <0xA0410000 0xFFFF>;
+			interrupts = <25 6>;
+			interrupt-parent = <&clic0>;
+			#address-cells = <1>;
+			#size-cells = <0>;
+			status = "disabled";
+		};
+
+		spi0: spi@0xA0800000 {
+			compatible = "andestech,atcspi200";
+			reg = <0xA0800000 0xFFFF>;
+			interrupts = <24 5>;
+			interrupt-parent = <&clic0>;
+			dmas = <&dma0 0 0 0x009>,
+			       <&dma0 1 1 0x00A>;
+			dma-names = "tx", "rx";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			clock-frequency = <66000000>;
+			status = "disabled";
+		};
+
+		dma0: dma@0xA0600000 {
+			compatible = "andestech,atcdmac300";
+			reg = <0xA0600000 0xFFFF>;
+			interrupts = <27 9>;
+			interrupt-parent = <&clic0>;
+			dma-channels = <8>;
+			dma-requests = <16>;
+			chain-transfer = <1>;
+			#dma-cells = <3>;
+			status = "disabled";
+		};
+
+		bcpu_wdt: wdt@0xA0120000 {
+			compatible = "andestech,atcwdt100";
+			reg = <0xA0120000 DT_SIZE_K(16)>;
+			clock-frequency = <266666667>;
+			intr-time-scale = <7>;
+			rst-time-scale = <7>;
+			interrupts = <0xffffffff 1>;
+			interrupt-parent = <&clic0>;
+			status = "okay";
+		};
+
+		pufcc: pufcc@0xA0130000 {
+			compatible = "pufcc";
+			reg = <0xA0130000 DT_SIZE_K(64)>;
+			interrupts = <33 3>;
+			interrupt-parent = < &clic0 >;	
+			status = "okay";		
+		};
+
+		fpga_cfg: cfg-ctrl@0xA0710000 {
+			compatible = "rapidsi,rigel";
+			reg = <0xA0710000 DT_SIZE_K(64)>;
+			#address-cells = <1>;
+			#size-cells = <0>;
+
+			ofe: ofe@0xA0710000 {
+				compatible = "rapidsi,ofe", "rapidsi,rigel";
+				reg = <0xA0710000>;
+				pinctrl-0 = <&ofe_default_pinmux>;
+				status = "disabled";
+			};
+
+			fcb: fcb@0xA0712000 {
+				compatible = "rapidsi,fcb", "rapidsi,rigel";
+				reg = <0xA0712000>;
+				status = "disabled";
+			};
+
+			icb: icb@0xA0716000 {
+				compatible = "rapidsi,icb", "rapidsi,rigel";
+				reg = <0xA0716000>;
+				status = "disabled";
+			};
+
+			pcb: pcb@0xA0718000 {
+				compatible = "rapidsi,pcb", "rapidsi,rigel";
```

Comment:
There are two compatibles. Either one can work. We have the specs defined for rapidsi,rigel in rapidsi,fpga.yaml. That is where i have also put the bindings for ofe since thats part of the fabric interface.